src/node.rs

```diff
@@ -803,53 +803,53 @@ impl<S: ForkSource + std::fmt::Debug> InMemoryNode<S> {
 
         let bytes_published = spent_on_pubdata / debug.gas_per_pubdata.as_u32();
 
-        println!(
+        log::info!(
             "During execution published {} bytes to L1, @{} each - in total {} gas",
             to_human_size(bytes_published.into()),
             to_human_size(debug.gas_per_pubdata),
             to_human_size(spent_on_pubdata.into())
         );
 
-        println!("Out of {} gas used, we spent:", to_human_size(gas_used));
-        println!(
+        log::info!("Out of {} gas used, we spent:", to_human_size(gas_used));
+        log::info!(
             "  {:>15} gas ({:>2}%) for transaction setup",
             to_human_size(intrinsic_gas),
             to_human_size(intrinsic_gas * 100 / gas_used)
         );
-        println!(
+        log::info!(
             "  {:>15} gas ({:>2}%) for bytecode preparation (decompression etc)",
             to_human_size(debug.gas_spent_on_bytecode_preparation),
             to_human_size(debug.gas_spent_on_bytecode_preparation * 100 / gas_used)
         );
-        println!(
+        log::info!(
             "  {:>15} gas ({:>2}%) for account validation",
             to_human_size(gas_for_validation),
             to_human_size(gas_for_validation * 100 / gas_used)
         );
-        println!(
+        log::info!(
             "  {:>15} gas ({:>2}%) for computations (opcodes)",
             to_human_size(gas_spent_on_compute),
             to_human_size(gas_spent_on_compute * 100 / gas_used)
         );
 
-        println!(
+        log::info!(
             "\n\n {}",
```

Comment:
To avoid having lines that don't start with logger information, should we replace all instances of manually adding a newline (`\n`) with `log::info!("");`?

BEFORE:
```rust
        log::info!(
            "\n\n {}",
            "=== Transaction setup cost breakdown ===".to_owned().bold(),
        );
```

AFTER:
```rust
        log::info!("");
        log::info!("");
        log::info!(
            " {}",
            "=== Transaction setup cost breakdown ===".to_owned().bold(),
        );
```